src/components/Buttons/ContainerPreviousIdea.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import { Button } from '@mui/material';
+
+const ContainerPreviousIdea = (onClick, disabled) => {
```

Comment:
What's the difference between `ContainerNextIdea` and `ContainerPreviousIdea` components? They look very similar and likely introduce unnecessary code duplication

Reply:
I can't explain my actions. They are the same! Fixed.